frontend/talentsearch/resources/js/components/workExperience/WorkExperience.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { useWatch } from "react-hook-form";
+import { Input } from "@common/components/form/Input";
+import { Checkbox } from "@common/components/form/Checkbox";
+import { errorMessages } from "@common/messages";
+
+export const WorkExperience: React.FunctionComponent = () => {
```

Comment:
Can you change the title of the component to WorkExperienceForm, just to keep with the convention we have. Also, please change the file name to `WorkExperienceForm.tsx`, so it matches.

Reply:
You'll also have to change the filename in storybook to `WorkExperienceForm.stories.tsx`, and it's contents to match.